.github/workflows/deploy-sepolia-vault-governance.yaml

```
@@ -0,0 +1,54 @@
+    steps:
+      - uses: actions/checkout@master
+        with:
+          fetch-depth: 0
+          submodules: recursive
+      - uses: foundry-rs/foundry-toolchain@v1
+      - run: forge install
+      - run: forge build
+      - run: forge tree
+      - run: forge script script/VaultGovernanceDeploy.s.sol:VaultGovernanceDeploy --rpc-url $RPC_URL --broadcast --gas-price  500000000000   --verify --verbosity 4
+        env:
+          RPC_URL:  ${{ secrets.RPC_URL }}
```

yamllint (1.29.0-1)

[warning] too many spaces after colon

(colons)